config/rm/settings.ini

```diff
@@ -17,6 +17,11 @@ pa.rm.local.nodes.number=-1
 # ping frequency used by node source for keeping a watch on handled nodes (in ms)
 pa.rm.node.source.ping.frequency=45000
 
+# Periodic down and lost nodes removal attempts (cron expression)
+pa.rm.nodes.unavailable.removal.frequency=* * * * *
```

Comment:
every minute check and 1 minute unavailable timeout seem far too low.

Down nodes should not be removed fast to allow actions by policies